src/resources/views/base/dashboard.blade.php

```diff
@@ -1,14 +1,21 @@
 @extends(backpack_view('blank'))
 
 @php
-    $widgets['before_content'][] = [
-        'type'        => 'jumbotron',
-        'heading'     => trans('backpack::base.welcome'),
-        'content'     => trans('backpack::base.use_sidebar'),
-        'button_link' => backpack_url('logout'),
-        'button_text' => trans('backpack::base.logout'),
-    ];
+    if (config('backpack.base.show_getting_started')) {
+        $widgets['before_content'][] = [
+            'type'        => 'view',
+            'view'        => 'backpack::inc.getting_started',
+        ];
+    } else {
+        $widgets['before_content'][] = [
+            'type'        => 'jumbotron',
+            'heading'     => trans('backpack::base.welcome'),
+            'content'     => trans('backpack::base.use_sidebar'),
+            'button_link' => backpack_url('logout'),
+            'button_text' => trans('backpack::base.logout'),
+        ];
+    }
```

Comment:
@pxpm heads-up, I am merging this later today. Last chance to stop me from doing something potentially stupid here 😅